packages/vue/src/examples/pages/product/Product.vue

```diff
@@ -472,194 +330,198 @@ export default {
     };
   },
   methods: {
-    toggleWishlist(index) {
-      this.products[index].isOnWishlist = !this.products[index].isOnWishlist;
+    addToCart() {
+      console.log(
+        `${this.qty} x ${this.product.name} (size: ${this.selectedSize}, color: ${this.selectedColor}) has been added to cart`
+      );
+    },
+    selectColor(colorIndex) {
+      this.product.colors.map((el, i) => {
+        if (colorIndex === i) {
+          el.selected = true;
+          this.selectedColor = el.name;
+        } else {
+          el.selected = false;
+        }
+      });
     }
   }
 };
 </script>
 <style lang="scss" scoped>
 @import "~@storefront-ui/vue/styles";
+
 #product {
   box-sizing: border-box;
   @include for-desktop {
-    max-width: 1240px;
+    max-width: 77.5rem;
     margin: 0 auto;
   }
 }
-.section {
-  padding: 0 var(--spacer-big);
-  @include for-desktop {
-    padding: 0;
-  }
-}
-.breadcrumbs {
-  padding: var(--spacer-big) var(--spacer-extra-big) var(--spacer-extra-big)
-    var(--spacer-extra-big);
-}
+
 .product {
-  @include for-desktop {
-    display: flex;
+  &__chevron {
+    --chevron-size: 14px;
+    position: absolute;
+    top: 0;
+    left: 0;
+    z-index: 1;
+    margin: var(--spacer-base) 0 0 var(--spacer-base);
   }
-  &__gallery,
-  &__description {
-    flex: 1;
+  &__images-desktop {
+    display: flex;
+    flex-direction: column;
   }
-  &__description {
-    padding: 0 var(--spacer-big);
+  &__info {
+    margin: var(--spacer-sm) auto var(--spacer-xs);
     @include for-desktop {
-      margin: 0 0 0 calc(var(--spacer-big) * 5);
+      max-width: 32.625rem;
+      margin: 0 0 0 7.5rem;
     }
-  }
-}
-.product-property {
-  margin: var(--spacer) 0;
-}
-.product-details {
-  &__heading {
-    --heading-title-font-size: var(--font-size-big);
-    --heading-title-font-weight: var(--body-font-weight-primary);
-    margin: var(--spacer-big) 0 0 0;
-    @include for-desktop {
-      --heading-title-font-size: var(--h1-font-size);
-      --heading-title-font-weight: var(--body-font-weight-secondary);
-      margin: 0;
-    }
-  }
-  &__sub {
-    @include for-desktop {
+    &__header {
```

Comment:
@justyna-13 maybe we should create less nested elements. Maybe we can change some elements to blocks 👍